source/documentation/archive/serving_data/fuseki1.md

```diff
@@ -0,0 +1,450 @@
+
+The Fuseki server can be given an
+[assembler description](/documentation/assembler/)
+to build a variety of model and datasets types.
+
+    fuseki-server --desc assembler.ttl /ds
+
+Full details of setting up models assembler is given in the
+[assembler documentation](/documentation/assembler/) and [assembler howto](/documentation/assembler/assembler-howto.html).
+
+A general dataset is described by:
+
+    # Dataset of default graph and one named graph.
+    <#dataset> rdf:type ja:RDFDataset ;
+       ja:defaultGraph <#modelDft> ;
+       ja:namedGraph
+           [ ja:graphName      <http://example.org/name1> ;
+             ja:graph          <#model1> ] ;
+       .
+
+    <#modelDft> a ja:MemoryModel ;
+            ja:content [ ja:externalContent <file:Data.ttl> .
+
+    <#model1>  rdf:type ja:MemoryModel ;
+       ja:content [ ja:externalContent <file:FILE-1.ttl> ] ;
+       ja:content [ ja:externalContent <file:FILE-2.ttl> ] ;
+       .
+
+The models can be
+[Jena inference models](/documentation/inference/).
+
+## Fuseki Configuration File
+
+A Fuseki server can be set up using a configuration file. The
+command-line arguments for publishing a single dataset are a short
+cut that, internally, builds a default configuration based on the
+dataset name given.
+
+The configuration is an RDF graph. One graph consists of one server
+description, with a number of services, and each service offers a
+number of endpoints over a dataset.
+
+The example below is all one file (RDF graph in Turtle syntax)
+split to allow for commentary.
+
+### Prefix declarations
+
+Some useful prefix declarations:
+
+    @prefix fuseki:  <http://jena.apache.org/fuseki#> .
+    @prefix rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+    @prefix rdfs:    <http://www.w3.org/2000/01/rdf-schema#> .
+    @prefix tdb:     <http://jena.hpl.hp.com/2008/tdb#> .
+    @prefix ja:      <http://jena.hpl.hp.com/2005/11/Assembler#> .
+    @prefix :        <#> .
+
+### Server Section
+
+Order of the file does not matter to the machine, but it's useful
+to start with the server description, then each of the services
+with its datasets.
+
+    [] rdf:type fuseki:Server ;
+       # Server-wide context parameters can be given here.
+       # For example, to set query timeouts: on a server-wide basis:
+       # Format 1: "1000" -- 1 second timeout
+       # Format 2: "10000,60000" -- 10s timeout to first result, then 60s timeout to for rest of query.
+       # See java doc for ARQ.queryTimeout
+       # ja:context [ ja:cxtName "arq:queryTimeout" ;  ja:cxtValue "10000" ] ;
+
+       # Services available.  Only explicitly listed services are configured.
+       #  If there is a service description not linked from this list, it is ignored.
+       fuseki:services (
+         <#service1>
+         <#service2>
+       ) .
+
+### Assembler Initialization
+
+All datasets are described by
+[assembler descriptions](../assembler/index.html).
```

Comment:
The links above to assembler documentation work as `/documentation/assembler/`, but this one gives 404 due to the extra `/archive/` in our new URL hierarchy.